modules/rosa-cluster-autoscaler-cli-settings.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-cli-autoscale-settings_{context}"]
+= Cluster autoscaling parameters using the ROSA CLI
+
+There are several parameters you can add to the cluster creation command to adjust autoscaling options. The table explains all the configurable autoscaler parameters when using the ROSA CLI (`rosa`).
+
+[cols="4",options="header"]
+|===
+|Setting
+|Description
+|Type or Range
+|Example/Instruction
+
+|`--autoscaler-balance-similar-node-groups`
+|Identify node groups with the same instance type and label set, and aim to balance respective sizes of those node groups.
+|`boolean`
+|Add it to set to true, omit the option to set to false.
+
+|`--autoscaler-skip-nodes-with-local-storage`
+|If set, cluster autoscaler does not delete nodes with pods that have local storage, for example, EmptyDir or HostPath.
+|`boolean`
+|Add it to set to true, omit the option to set to false.
+
+|`--autoscaler-log-verbosity _int_`
+|Autoscaler log level. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-log-verbosity 4`
+
+|`--autoscaler-max-pod-grace-period _int_`
+|Gives pods graceful termination time before scaling down, measured in seconds. Replace _int_ in the command with the number you want to use.
+|`integer`
+|`--autoscaler-max-pod-grace-period 0`
+
+|`--autoscaler-pod-priority-threshold _int_`
+|The priority that a pod must exceed to cause the cluster autoscaler to deploy additional nodes. Replace _int_ in the command with the number you want to use, can be negative.
+|`integer`
+|`--autoscaler-pod-priority-threshold -10`
+
+|`autoscaler-gpu-limit`
+|Minimum and maximum number of different GPUs in cluster. Cluster autoscaler does not scale the cluster beyond these numbers. The format should be a comma-separated list of "<gpu_type>:<min>:<max>".
```

Comment:
the format is not a comma-separated list of "<gpu_type>:<min>:<max>", but rather this option should be repeated to apply more restrictions for more GPU types

Reply:
and in the case of the CLI it's separated by a comma between the values, i.e. ``<gpu_type>,<min>,<max>``